frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -37,6 +38,8 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
   const [storageClass, setStorageClass] = React.useState('');
   const [inProgress, setProgress] = React.useState(false);
   const [errorMessage, setError] = React.useState('');
+  const [isNoProvisionerSC, setNoProvisionerSC] = React.useState(false);
```

Comment:
Consider another approach for getting this information. Changing `storageClass` data from `string` to `K8sCommon` should do the trick.

Reply:
Why do you think another approach is required here?

Reply:
https://github.com/openshift/console/pull/5016/files#r407903025